stream_alert/alert_processor/outputs/carbonblack.py

```diff
@@ -0,0 +1,108 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from collections import OrderedDict
+from cbapi.response import BannedHash, Binary, CbResponseAPI
+from stream_alert.alert_processor import LOGGER
+
+from stream_alert.alert_processor.outputs.output_base import (
+    OutputDispatcher,
+    OutputProperty,
+    StreamAlertOutput
+)
```

Comment:
Can we add spacing so that the imports look like this:

```python
from collections import OrderedDict

from cbapi.response import BannedHash, Binary, CbResponseAPI

from stream_alert.alert_processor import LOGGER
from stream_alert.alert_processor.outputs.output_base import (
    OutputDispatcher,
    OutputProperty,
    StreamAlertOutput
)
```

Reply:
Update will be pushed shortly